Makefile.psx.mk

```diff
@@ -4,424 +4,258 @@
-	./tools/splat_ext/animset.py gen-asm $< $(BUILD_DIR)/$(ASSETS_DIR)/weapon/$*_1.animset.s -s g_Animset
-	$(AS) $(AS_FLAGS) -o $@ $(BUILD_DIR)/$(ASSETS_DIR)/weapon/$*_1.animset.s
-$(BUILD_DIR)/$(ASSETS_DIR)/weapon/%_2.animset.o: $(ASSETS_DIR)/weapon/%_2.animset.json
-	./tools/splat_ext/animset.py gen-asm $< $(BUILD_DIR)/$(ASSETS_DIR)/weapon/$*_2.animset.s -s g_Animset2
-	$(AS) $(AS_FLAGS) -o $@ $(BUILD_DIR)/$(ASSETS_DIR)/weapon/$*_2.animset.s
-
-# assembly and c files
-$(BUILD_DIR)/%.s.o: %.s
-	mkdir -p $(dir $@)
-	$(AS) $(AS_FLAGS) -o $@ $<
-
-$(BUILD_DIR)/%.c.o: %.c $(MASPSX_APP) $(CC1PSX)
-	mkdir -p $(dir $@)
-	$(CPP) $(CPP_FLAGS) -lang-c $< | $(SOTNSTR) | $(ICONV) | $(CC) $(CC_FLAGS) $(PSXCC_FLAGS) | $(MASPSX) | $(AS) $(AS_FLAGS) -o $@
-
-$(BUILD_DIR)/$(SRC_DIR)/main/psxsdk/libgpu/sys.c.o: $(SRC_DIR)/main/psxsdk/libgpu/sys.c $(MASPSX_APP) $(CC1PSX)
-	$(CPP) $(CPP_FLAGS) -lang-c $< | $(SOTNSTR) | $(ICONV) | $(CC) $(CC_FLAGS) $(PSXCC_FLAGS) | $(MASPSX_21) | $(AS) $(AS_FLAGS) -o $@
-
-extract_assets: $(SOTNASSETS)
-	cd tools/sotn-assets; $(GO) install
-	$(SOTNASSETS) extract config/assets.$(VERSION).yaml
-build_assets: $(SOTNASSETS)
-	$(SOTNASSETS) build config/assets.$(VERSION).yaml
-
-$(BUILD_DIR)/assets/dra/memcard_%.png.o: assets/dra/memcard_%.png
-	mkdir -p $(dir $@)
-	$(PNG2S) encode $< \
-		$(BUILD_DIR)/assets/dra/memcard_$*.png.s g_saveIcon$* \
-		$(BUILD_DIR)/assets/dra/memcard_$*.pal.s g_saveIconPal$*
-	$(AS) $(AS_FLAGS) -o $(BUILD_DIR)/assets/dra/memcard_$*.png.o $(BUILD_DIR)/assets/dra/memcard_$*.png.s
-	rm $(BUILD_DIR)/assets/dra/memcard_$*.png.s
-	$(AS) $(AS_FLAGS) -o $(BUILD_DIR)/assets/dra/memcard_$*.pal.o $(BUILD_DIR)/assets/dra/memcard_$*.pal.s
-	rm $(BUILD_DIR)/assets/dra/memcard_$*.pal.s
-$(BUILD_DIR)/assets/st/sel/memcard_%.png.o: assets/st/sel/memcard_%.png
-	mkdir -p $(dir $@)
-	$(PNG2S) encode $< \
-		$(BUILD_DIR)/assets/st/sel/memcard_$*.png.s g_saveIcon$* \
-		$(BUILD_DIR)/assets/st/sel/memcard_$*.pal.s g_saveIconPal$*
-	$(AS) $(AS_FLAGS) -o $(BUILD_DIR)/assets/st/sel/memcard_$*.png.o $(BUILD_DIR)/assets/st/sel/memcard_$*.png.s
-	rm $(BUILD_DIR)/assets/st/sel/memcard_$*.png.s
-	$(AS) $(AS_FLAGS) -o $(BUILD_DIR)/assets/st/sel/memcard_$*.pal.o $(BUILD_DIR)/assets/st/sel/memcard_$*.pal.s
-	rm $(BUILD_DIR)/assets/st/sel/memcard_$*.pal.s
-
+WEAPON0_FILES 	:= $(foreach num,$(shell seq -w 000 058),$(BUILD_DIR)/weapon/f0_$(num).bin $(BUILD_DIR)/weapon/w0_$(num).bin)
+WEAPON1_FILES 	:= $(foreach num,$(shell seq -w 000 058),$(BUILD_DIR)/weapon/f1_$(num).bin $(BUILD_DIR)/weapon/w1_$(num).bin)
+
+DRA_SYMBOLS 	:= $(BASE_SYMBOLS) $(CONFIG_DIR)/symbols.$(VERSION).%.txt
+STMAD_SYMBOLS 	:= $(CONFIG_DIR)/symbols.beta.txt $(CONFIG_DIR)/symbols.stmad.txt
+WEAPON_SYMBOLS 	:= $(CONFIG_DIR)/symbols.$(VERSION).weapon.txt $(CONFIG_DIR)/symbols.$(VERSION).weapon.txt.in
+STAGE_SYMBOLS 	:= $(if $$(filter mad,%),$(STMAD_SYMBOLS),$(BASE_SYMBOLS) $(CONFIG_DIR)/symbols.$(VERSION).st%.txt)
```

Comment:
I just discovered that this isn't working as I thought it was, it is actually working the way I thought it was supposed to.
I will be reworking this line, but it isn't an issue for now since it is referencing static files that are expected to be present from the repo.